tap_hubspot/__init__.py

```diff
@@ -229,13 +230,17 @@ def parse_source_from_url(url):
                       on_giveup=on_giveup,
                       factor=2)
 @utils.ratelimit(9, 1)
-def request(url, params=None):
+def request(url, params={}):
```

Comment:
This is a common "gotcha" in Python, but default mutable arguments are bad.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Reply:
Thanks -- I've been away from Python for long enough that I'd forgotten about that.